EssentialCSharp.ListingManager.sln

```diff
@@ -11,7 +11,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 	ProjectSection(SolutionItems) = preProject
 		Directory.Build.props = Directory.Build.props
 		Directory.Packages.props = Directory.Packages.props
-		NuGet.config = NuGet.config
+		Nuget.config = Nuget.config

```

Comment:
Why the casing change here? The file looks to have a capital G when I view the repo.